addons/aldryn-django/aldryn_config.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+import os
+import dj_database_url
+import django_cache_url
+import warnings
+from aldryn_addons.utils import boolean_ish
+from aldryn_client import forms
+from getenv import env
+
+
+class Form(forms.BaseForm):
+    name = forms.CheckboxField('Name', required=False)
+
+    def to_settings(self, data, settings):
+        # TODO: once we have proper "secrets" support, these should not load
+        #       from the env themselves anymore
+        # TODO: some sort of warnings mechanism for required settings where
+        #       defaults have been set (like SECRET_KEY)
+        settings['BASE_DIR'] = settings.get(
+            'BASE_DIR',
+            os.path.dirname(os.path.abspath(settings['__file__']))
+        )
+        settings['DATA_ROOT'] = env('DATA_ROOT', os.path.join(settings['BASE_DIR'], 'data'))
+        settings['SECRET_KEY'] = env('SECRET_KEY', 'this-is-not-very-random')
+        settings['DEBUG'] = boolean_ish(env('DEBUG', False))
+        settings['TEMPLATE_DEBUG'] = boolean_ish(env('DEBUG', settings['DEBUG']))
+
+        settings['DATABASE_URL'] = env('DATABASE_URL')
+        if settings['DATABASE_URL']:
+            pass
+        elif env('DJANGO_MODE') == 'build':
+            settings['DATABASES']['default'] = {
+                'ENGINE': 'django.db.backends.sqlite3',
+                'NAME': ':memory:',
+            }
+        else:
+            settings['DATABASE_URL'] = 'sqlite:///{}'.format(
+                os.path.join(settings['DATA_ROOT'], 'db.sqlite3')
+            )
+            warnings.warn(
+                'no database configured. Falling back to DATABASE_URL={0}'.format(
+                    settings['DATABASE_URL']
+                ),
+                RuntimeWarning,
+            )
+        settings['DATABASES']['default'] = dj_database_url.parse(settings['DATABASE_URL'])
+
+        settings['ALLOWED_HOSTS'] = env('ALLOWED_HOSTS', ['*'])
```

Comment:
let's use ``ALLOWED_HOSTS = ['localhost', '*']`` instead. I had issues before where Django checked ``if 'localhost' in ALLOWED_HOSTS`` and failed because of that